org.osgi.test.assertj.framework/src/test/java/org/osgi/test/assertj/test/testutil/ConditionAssert.java

```diff
@@ -0,0 +1,75 @@
+/*******************************************************************************
+ * Copyright (c) Contributors to the Eclipse Foundation
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ *******************************************************************************/
+
+package org.osgi.test.assertj.test.testutil;
+
+import static org.assertj.core.api.Assertions.assertThatThrownBy;
+
+import java.util.regex.Pattern;
+
+import org.assertj.core.api.AbstractThrowableAssert;
+import org.assertj.core.api.Condition;
+import org.assertj.core.api.ObjectAssertFactory;
+
+public interface ConditionAssert {
```

Comment:
Isn't this class in another PR?

Reply:
yes. the first merged PR get it in wins. will remove it from the other PRs